From fa134cdd6a7bfc0841db0b2f028e0a5c1c144f2c Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 12 Nov 2018 21:59:24 -0700
Subject: [PATCH 05/18] msm8974-common: sepolicy: Label sysfs_disk_stat nodes

* avc: denied { read } for name="stat" dev="sysfs" ino=26461
  scontext=u:r:storaged:s0 tcontext=u:object_r:sysfs:s0 tclass=file
  permissive=1

Change-Id: I4b7258d069801f542da8c7f5ca8242ea32f12bca
---
 sepolicy/common/file_contexts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/sepolicy/common/file_contexts b/sepolicy/common/file_contexts
index 2490cfe..c8b3dfa 100644
--- a/sepolicy/common/file_contexts
+++ b/sepolicy/common/file_contexts
@@ -43,6 +43,9 @@
 /sys/devices/i2c\.[0-9]+/i2c-[0-9]+/[0-9]+-[a-z0-9]+/power_supply(/.*)?                      u:object_r:sysfs_batteryinfo:s0
 /sys/devices/msm_dwc3/power_supply(/.*)?                u:object_r:sysfs_batteryinfo:s0
 
+# sysfs - block/storage
+/sys/devices/msm_sdcc\.[0-9]/mmc_host/mmc[0-9]/mmc[0-9]:[0-9]+/block/mmcblk[0-9]/stat        u:object_r:sysfs_disk_stat:s0
+
 # sysfs - graphics/panel
 /sys/class/mhl(/.*)?                                    u:object_r:sysfs_graphics:s0
 /sys/devices/mdp\.[0-9](/.*)?                           u:object_r:sysfs_graphics:s0
-- 
2.17.1

